linux-gcc-jetson-tk1.cmake

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2017, NeroBurner
+# All rights reserved.
+
+# install cross compiler on Ubuntu
+# - sudo apt install g++-arm-linux-gnueabihf
+# - with gfortran: sudo apt install gfortran-arm-linux-gnueabihf
+
+if(DEFINED POLLY_LINUX_GCC_JETSON_TK1_)
+  return()
+else()
+  set(POLLY_LINUX_GCC_JETSON_TK1_ 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_init.cmake")
+
+polly_init(
+    "Linux / gcc / armhf / c++11 support / neon-vfpv4 / cortex-a15"
+    "Unix Makefiles"
+)
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_common.cmake")
+
+# set system name, this sets the variable CMAKE_CROSS_COMPILING
+set(CMAKE_SYSTEM_NAME Linux)
+set(CROSS_COMPILE_TOOLCHAIN_PREFIX "arm-linux-gnueabihf")
+set(CMAKE_CROSSCOMPILING_EMULATOR qemu-arm) # used for try_run calls
+
+include(
+    "${CMAKE_CURRENT_LIST_DIR}/compiler/gcc-cross-compile-simple-layout.cmake"
+)
+include("${CMAKE_CURRENT_LIST_DIR}/flags/cxx11.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/flags/hardfloat.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/flags/neon-vfpv4.cmake")
+
+polly_add_cache_flag(CMAKE_C_FLAGS "-mtune=cortex-a15")
```

Comment:
Looks like this flag doesn't have any effect on any of these macro:
* https://github.com/ruslo/hunter/blob/master/scripts/predefined.list

Meaning `linux-gcc-jetson-tk1` toolchain will have same `Toolchain-ID` as `linux-gcc-armhf-neon-vfpv4`. If there is such macro then it should be added to Hunter, if there is no such macro you should modify `HUNTER_TOOLCHAIN_UNDETECTABLE_ID`:
* https://github.com/ruslo/polly/blob/4556dfd404a2f24da9edc10a537b74c1bf178c3c/flags/sanitize_leak.cmake#L18

Also I think it make sense to move this flag to separate module as you did with `hardfloat` and `neon-vfpv4`.

Reply:
I did not find a macro that gets defined with setting mtune. Added it to the undetectable flags.

what name would you choose for the undetectable flag? I can't decide between the following two:
- `mtune_cortex-a15`
- `cortex-a15`

Reply:
> I did not find a macro that gets defined with setting mtune

Okay, but it's strange. What if I want to make an assembly insertion (rhetorical question)?

> cortex-a15

This one sounds good. The only requirement is that it will not conflict with existent `HUNTER_TOOLCHAIN_UNDETECTABLE_ID`.